x11-xserver-utils.mk

```diff
@@ -0,0 +1,47 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS      += x11-xserver-utils
+X11_XSERVER_UTILS_VERSION := 7.7+8
+DEB_X11_XSERVER_UTILS_V   ?= $(X11_XSERVER_UTILS_VERSION)
+
+x11-xserver-utils-setup: setup
+	wget -q -nc -P$(BUILD_SOURCE) https://salsa.debian.org/xorg-team/app/x11-xserver-utils/-/archive/x11-xserver-utils-$(X11_XSERVER_UTILS_VERSION)/x11-xserver-utils-x11-xserver-utils-$(X11_XSERVER_UTILS_VERSION).tar.gz
+	$(call EXTRACT_TAR,x11-xserver-utils-x11-xserver-utils-$(X11_XSERVER_UTILS_VERSION).tar.gz,x11-xserver-utils-x11-xserver-utils-$(X11_XSERVER_UTILS_VERSION),x11-xserver-utils)
+
+ifneq ($(wildcard $(BUILD_WORK)/x11-xserver-utils/.build_complete),)
+x11-xserver-utils:
+	@echo "Using previously built x11-xserver-utils."
+else
+x11-xserver-utils: x11-xserver-utils-setup libsm libx11 libxaw libxcursor libxext libxft libxmu libxrender libxt libpng16 libxkbfile xbitmaps
+	+for dir in $(BUILD_WORK)/x11-xserver-utils/*; do \
+		if [ -f $$dir/configure ]; then \
```

Comment:
This if statement doesn't need to be that long. It should just generate the configure with `autoreconf` if it doesn't already exist, then end there.

Reply:
2 packages would fail to compile without the autoconf